docs/Doxyfile-mcss.in

```diff
@@ -0,0 +1,14 @@
+@INCLUDE                = Doxyfile-@BLF_TAG@
+
+USE_MDFILE_AS_MAINPAGE  = "@BLF_MAIN_PAGE@"
```

Comment:
```suggestion
USE_MDFILE_AS_MAINPAGE  = "@IDYNTREE_MAIN_PAGE@"
```

However I don't know if you use this parameter

Reply:
I use this parameter, but if I change it here, I need to also change it in `docs/generate_documentation_files.py`, with little added benefit as the behaviour of the scripts will be exactly the same. In general, I would like to avoid any unnecessary change to those scripts, in view of eventually unifying the scripts, or at the minimum port changes from iDynTree to BLF or from BLF to iDynTree.

Reply:
Given that I think you can merge it 😄